aliquotmaf/subcommands/vcf_to_aliquot/extractors/genotypes.py

```diff
@@ -85,6 +81,34 @@ def extract(cls, var_allele_idx, genotype, alleles):
             bcount = list(genotype["BCOUNT"])
             depths = [bcount[b_idx[i]] if i in b_idx else None for i in alleles]
 
+        # Handle CaVEMan which provides genotype and counts for all nucleotides in forward and reverse strands
+        elif set(
+            ["GT", "FAZ", "FCZ", "FGZ", "FTZ", "RAZ", "RCZ", "RGZ", "RTZ"]
+        ).issubset(genotype.keys()):
```

Comment:
Just making sure: is `genotype.keys()` is supposed to be the superset here?

Reply:
I could probably set it to be equal, this is not the best way of detecting whether this is caveman but of the callers we run I don't think it'd cause a problem

Reply:
actually, I guess I'm just not 100% sure that `genotype` doesn't get some additional keys so I'd rather just leave it as subset